### PR TITLE
add sql files

### PR DESCRIPTION
**Pull request by medlamrani**
- [ ] add sql files

### CODE REVIEW

File: db/00_create-table.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+
+CREATE TABLE IF NOT EXISTS `post` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `author` varchar(255) NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `contain` text NOT NULL,
```
Comment (medlamrani) [inline db/00_create-table.sql]: content

File: db/00_create-table.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  `addDate` datetime NOT NULL,
+  `updateDate` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=15 ;
+
+CREATE TABLE IF NOT EXISTS `comments` (
```
Comment (medlamrani) [inline db/00_create-table.sql]: comment

File: db/00_create-table.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
+
+CREATE TABLE IF NOT EXISTS `post` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `author` varchar(255) NOT NULL,
```
Comment (medlamrani) [inline db/00_create-table.sql]: author for user id

File: db/00_create-table.sql [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  `addDate` datetime NOT NULL,
+  `updateDate` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=15 ;
+
+CREATE TABLE IF NOT EXISTS `comments` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `postId` int(11) NOT NULL,
+  `author` varchar(255) NOT NULL,
+  `contain` text NOT NULL,
```
Comment (medlamrani) [inline db/00_create-table.sql]: content

File: db/00_create-table.sql [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  `title` varchar(255) NOT NULL,
+  `contain` text NOT NULL,
+  `addDate` datetime NOT NULL,
+  `updateDate` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=15 ;
+
+CREATE TABLE IF NOT EXISTS `comments` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `postId` int(11) NOT NULL,
```
Comment (medlamrani) [inline db/00_create-table.sql]: post_id

File: db/00_create-table.sql [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  `contain` text NOT NULL,
+  `report` int(11) NOT NULL,
+  `commentDate` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=13 ;
+
+
+
+
+CREATE TABLE IF NOT EXISTS `user` (
```
Comment (medlamrani) [inline db/00_create-table.sql:25]: unique key constraint for username  field

File: db/00_create-table.sql (hunk 2 above)
Comment (medlamrani) [reply db/00_create-table.sql]: done

File: controller/controller.php [hunk 7]
```diff
@@ -25,15 +25,15 @@ public function post()
         require('views/postView.php');
     }
 
-    public function addComment($postId)
+    public function addComment($post_id)
```
Comment (eddymio) [inline controller/controller.php]: Here camelCase, we are not in SQL

**Pull request merged**